osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -69,7 +69,9 @@ protected virtual void ParseLine(T output, Section section, string line)
 
         protected string StripComments(string line)
         {
-            var index = line.IndexOf("//", StringComparison.Ordinal);
+            const string comment_prefix = "//";
```

Comment:
not sure what the overhead of the string construction is, but if we're going this far, would `var index = line.AsSpan().IndexOf(new Span<char>(new [] { '/', '/' }));` possibly be an improvement?

Reply:
Can just be moved inlined due to string interning. Const is not really useful.